### PR TITLE
Two issues related to GAWB-1250

### PR DESCRIPTION
**Pull request by MattPutnam**
- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [x] **Submitter**: If you changed a URL that is used elsewhere (e.g. in an email), comment about where it is used and ensure the dependent code is updated.
- [x] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [x] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [x] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [x] **Submitter**: If you're adding new libraries, sign us up to security updates for them
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [x] **TL** sign off
- [x] **LR** sign off
- [x] **Product Owner** sign off
- [x] **Submitter**: Verify all tests go green, including CI tests
- [x] **Submitter**: Squash commits and merge to develop
- [x] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

**Comment by zarsky-broad** [comment]
Hmm, this appears to get rid of the loading indicator when switching entity types and waiting for the data to appear.

**Comment by MattPutnam** [comment]
I think it's still there, it just doesn't show up for 350ms.

**Comment by zarsky-broad** [comment]
Can we get rid of that timer and show it immediately? Otherwise it just says "no data" for that time, which is long enough to start to worry

**Comment by MattPutnam** [comment]
I can poke at it.

File: src/cljs/main/broadfcui/page/workspace/summary/tab.cljs [hunk 1]
```diff
@@ -172,6 +172,8 @@
                          :data-btm-anchor (str body-id ":bottom")}
           :contents
           [:div {:style {:width 270}}
+           (when-not (and library-schema billing-projects curator?)
+             (comps/render-blocker "Loading..." "absolute"))
```
Comment (MattPutnam) [inline src/cljs/main/broadfcui/page/workspace/summary/tab.cljs]: I don't like having to write "absolute" as a consumer of this API.

**Comment by zarsky-broad** [comment]
@MattPutnam better?

**Review by davidangb — APPROVED** [review]
:+1: as TL and interim PO

**Review by davidangb — APPROVED** [review]
👍 as TL and interim PO (again)

**Pull request merged**